docs/development.rst

```diff
@@ -1122,7 +1122,117 @@ to him until directed).
 Adding a DFE model
 ******************
 
-TODO: WRITE ME
+`stdpopsim` supports distribution of fitness effects (DFE) models from the 
+literature. To add a new DFE model to the catalog, you will need to:
+
+    1. `Fork the repository and create a branch`_
```

Comment:
I was linking the sections from the demographic model docs, but maybe shouldn't? It's kinda weird that it jumps all the way up to that section.